### PR TITLE
Kasper4

### PR DESCRIPTION
**Pull request by Qave**

### CODE REVIEW

**Review by Noitcereon — DISMISSED** [review]
Reminder: img and media-assets to be merged at some point (in public folder)

File: .env.example [hunk 1]
```diff
@@ -48,4 +48,4 @@ PUSHER_APP_SECRET=
 PUSHER_APP_CLUSTER=mt1
 
 MIX_PUSHER_APP_KEY="${PUSHER_APP_KEY}"
-MIX_PUSHER_APP_CLUSTER="${PUSHER_APP_CLUSTER}"
+MIX_PUSHER_APP_CLUSTER="${PUSHER_APP_CLUSTER}"
```
Comment (Lausius) [inline .env.example:51]: dafuq is this? :D

**Review by Lausius — DISMISSED** [review]
looks good :)

**Review by Lausius — APPROVED** [review]
had to fix merge conflict :)

File: .env.example (hunk 1 above)
Comment (Qave) [reply .env.example:51]: i have no fucking clue xD

**Review by Noitcereon — APPROVED** [review]
All good

**Pull request merged**